recipes-ros/control-toolbox/control-toolbox_1.15.0.bb

```diff
@@ -4,7 +4,7 @@ LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://package.xml;beginline=7;endline=7;md5=d566ef916e9dedc494f5f793a6690ba5"
 
 DEPENDS = "rosconsole tf roscpp angles message-generation dynamic-reconfigure libtinyxml \
-    realtime-tools message-filters"
+    realtime-tools message-filters control-msgs"
```

Comment:
On kinetic-experimental-v3-alpha1, this issue is already addressed with commit b5f8edda4cd8 ("control-toolbox: add control-msgs").

For the indigo release, the `control_toolbox` package does not need `control_msgs` [1]; commit b5f8edda4cd8 is specific to kinetic and does not need be backported to master.

[1] https://github.com/ros-controls/control_toolbox/blob/indigo-devel/package.xml